packages/utils/README.md

```diff
@@ -0,0 +1,10 @@
+utils
+------
+
+Utils to be shared across 0x projects and packages
+
+## Install
+
+```bash
+yarn add @0xproject/utils
+```
```

Comment:
There are **many** more util methods shared between our packages, such as `consoleLog` and `spawnSwitchErr`. Can you please go through all the util methods in `0x.js`, `website` and `contracts` and extract all duplicate methods into this package?

Reply:
This can be done in a subsequent PR.